mailbox/store/src/main/java/org/apache/james/mailbox/store/json/SimpleMailboxACLJsonConverter.java

```diff
@@ -34,7 +34,7 @@
 
     interface Rfc4314RightsMixIn {
         @JsonValue
-        int serializeAsInteger();
+        int serialize();
```

Comment:
still int ?

Reply:
Arch...

Reply:
Arch linux has some weakness but I suspect it has nothing to do with that bug :trollface: